.github/actions/report-ci-errors/README.md

```diff
@@ -0,0 +1,26 @@
+## Pre-requisites
+
+You will need to get the slack webhook url from the secrets manager first and add it as SLACK_WEBHOOK_URL as a repository secret. 
+
+## Usage example:
```

Comment:
The formatting is off here
<img width="1059" alt="Screenshot 2025-02-27 at 12 45 49" src="https://github.com/user-attachments/assets/392a1439-4d7c-4422-b64b-f8af3d425039" />